MetaMask/metamask-mobile
# fix: locks api spec version for api spec tests

## **Description**
Ideally the api-specs that is used should not be pointing at latest, it should be locked to a version. The downstream execution apis was updated and included in our spec and needs some fixes. In the meantime this locks the version to the previous one.


## **Related issues**

Fixes:

## **Manual testing steps**

1. run `yarn test:e2e:ios:debug:build`
2. run `yarn test:api-specs`
3. see that it passes

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
